### ISSUE TITLE
PostgreSQL 10 SCRAM authentication fails

### ISSUE THREAD

**Issue by ced75** [cybertec-postgresql/pgwatch2#77]
Hi,

I'm using the last 1.3.7 release of pgwatch2 in a Docker container.
I'm trying to configure a PostgreSQL server in v10, using SCRAM authentication.
But the connection fails. I tried a psql connection from inside the Docker container, but I get the following error:
psql: SCRAM authentication requires libpq version 10 or above

Is it possible to update libpq do version 10 in pgwatch2 ?

Thanks in advance,
Cedric

**Comment by kmoppel** [comment]
Hi! Indeed, would be nice to have it - queued :)

**Comment by ced75** [comment]
Thanks for all !

**Comment by kmoppel** [comment]
FYI - waiting on https://github.com/lib/pq/pull/608 ...

**Comment by kmoppel** [comment]
There as of v1.6.0: https://github.com/cybertec-postgresql/pgwatch2/blob/master/CHANGELOG.md#v160-2019-06-19
